stanozzy/stackoverflow
# 162558098 login page

*What does this PR do*

Builds the login page for users

*Description of task to be completed*

• Write HTML for the login page

• Write CSS to style the login page

*How should this be manually tested*

1. Clone this repository
2. Cd into the StackOverflow-lite folder
3. Open login.html from a browser

*What are the relevant pivotal tracker stories*

#162558098 A user should be able to log in to his/her StackOveflow-lite account

![screenshot from 2018-12-16 00-54-13](https://user-images.githubusercontent.com/45230437/50047767-e608e780-00b3-11e9-9b41-e659f1035973.png)
